javascript/ql/lib/semmle/javascript/frameworks/SQL.qll

```diff
@@ -539,4 +600,22 @@ private module SpannerCsv {
         ]
     }
   }
+
+  class SpannerSources extends ModelInput::SourceModelCsv {
+    string spannerClass() { result = ["v1.SpannerClient", "Database", "Transaction", "Snapshot",] }
+
+    string resultPath() {
+      result =
+        [
+          "Member[executeSql].Argument[0..].Parameter[1]",
+          "Member[executeSql].ReturnValue.Awaited.Member[0]", "Member[run].ReturnValue.Awaited",
+          "Member[run].Argument[0..].Parameter[1]",
+        ]
+    }
+
+    override predicate row(string row) {
+      row =
+        "@google-cloud/spanner;" + spannerClass() + ";" + resultPath() + ";database-access-result"
```

Comment:
```suggestion
        "@google-cloud/spanner;" + this.spannerClass() + ";" + this.resultPath() +
          ";database-access-result"
```
Fix code-scanning recommendation, (and an attempt to keep the auto-formatter happy).

Reply:
have done so :)